05_sumAll/sumAll.js

```diff
@@ -1,6 +1,22 @@
-const sumAll = function() {
-
+const sumAll = function (firstArg, secArg) {
```

Comment:
Of course they are arguments, try to find a name that would better describe type or usage of the value.

Reply:
Would "rangeStart" and "rangeEnd" be appropriate?